cmd/bosun/web/static/partials/rule.html

```diff
@@ -95,7 +95,7 @@
 </div>
 <div class="row" ng-show="error">
 	<div class="col-lg-12">
-		<div class="alert alert-danger" ng-bind="error"></div>
+		<pre class="alert alert-danger" ng-bind="error"></pre>
```

Comment:
true. the alternative is that we add `<br>` at every newline, i suppose. but afaik pre is always used for this sort of thing